eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -89,6 +96,12 @@ stages:
                           arguments: '-pathToArtifacts $(Package.Archive) -accessLevel "public" -tag $(Tag) -additionalTag "$(AdditionalTag)" -registry ${{parameters.Registry}} -npmToken $(azure-sdk-npm-token)'
                           pwsh: true
 
+                      - pwsh: |
+                          $result = eng/scripts/tools/remove-npm-next-tag.ps1 -packageArtifact $pkgArtifact -workingDirectory $(System.DefaultWorkingDirectory) -npmToken $(azure-sdk-npm-token)
```

Comment:
Is the result necessary here? If so should we be validating it?

Reply:
result is not required. I will remove it in next revision